plumpy/processes.py

```diff
@@ -80,7 +64,8 @@ def ensure_not_closed(func):
 
     @functools.wraps(func)
     def func_wrapper(self, *args, **kwargs):
-        if self._closed:  # pylint: disable=protected-access
+        # pylint: disable=protected-access, useless-suppression
```

Comment:
Is the `useless-suppression` that occurs here really a false-positive or can we just use the disable statement that is no longer needed?

Reply:
Reading it again, I think the `protected-access` is actually not needed and so it can be removed. That is why pylint issued the `useless-suppression`. So jsut remove both disables